about.html

```diff
@@ -10,54 +10,113 @@
   <link rel="stylesheet" href="css/index.css">
 
 </head>
-
+<header class='navbar'>
+    <img src= "lambda-black.png">
+      <nav class='nh'>
+        <div class='links'>
+           <a href='index.html'>Home</a>
+           <a href='about.html'>About</a>
+           <a href='products'>Products</a>
+           <a href='blog'>Blog</a>
+           <a href='contact'>Contact</a>
+        </div>
+      </nav>
+         <div class='jumbo'><img  src='jumbo-about.png' ></div> 
+    </header>
 <body>
     <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+   <section class='abouttop'>
```

Comment:
One other thing to remember about naming your css classes is that you should make them lowercase, which you do, and put dashes between words. 

Something like 

`about-top`